tmail-backend/mailbox/team-mailboxes/src/main/scala/com/linagora/tmail/team/TeamMailbox.scala

```diff
@@ -0,0 +1,26 @@
+package com.linagora.tmail.team
+
+import org.apache.james.core.Username
+import org.apache.james.mailbox.model.{MailboxConstants, MailboxPath}
+
+object Namespace {
+  val TEAM_MAILBOX_NAMESPACE: Namespace = Namespace(MailboxConstants.NAMESPACE_PREFIX_CHAR + "tm")
```

Comment:
`#tm` sounds too short to me, not very explicit.

In the ticket we defined it as: `#TeamMailbox`

After we can discuss should have everything lowercase or what not but `tm` does not mean much, IMO